boot/seal.go

```diff
@@ -141,20 +139,43 @@ func fallbackKeySealRequests(key, saveKey secboot.EncryptionKey) []secboot.SealK
 	}
 }
 
+type fdeSetupHookResult struct {
+	EncryptedKey []byte `json:"encrypted-key"`
```

Comment:
if we must keep sealed-key on reaveal, this name is strange.  We need to discuss what we can or can't change at this point.

Reply:
I reached out to ondra about this.